### PR TITLE
WIP: Information regarding raiden usage and contracts

### PR DESCRIPTION
**Pull request by agatsoh**
Gives some more information regarding raiden usage and deployed service contracts and associated tokens

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -45,6 +45,74 @@ the services follow these instructions:
 This installs the executables ``pathfinding-service``, ``monitoring-service``
 and ``request-collector`` into the virtual environment.
 
+Using MS and PFS in raiden
```
Comment (palango) [inline docs/index.rst]: While this is nice, doesn't this belong into the client docs?

File: docs/index.rst [hunk 2]
```diff
@@ -45,6 +45,74 @@ the services follow these instructions:
...
+
+Raiden client needs to be started with the raiden-services flags like
+
+* --enable-monitoring
+* --routing-mode [pfs|local|private]
+
+Doing ``raiden --help`` and going to Raiden Services options would give more
+information.
+
+Payments related to Services
```
Comment (palango) [inline docs/index.rst]: What target group is this written for? I'd say users don't care about the contracts...

File: docs/index.rst [hunk 3]
```diff
@@ -45,6 +45,74 @@ the services follow these instructions:
...
+.. _contracts: https://github.com/raiden-network/raiden-contracts
+  /blob/master/raiden_contracts/data_0.17.0/deployment_services_goerli.json
+
+
+The various *raiden-service* contracts and associated tokens can be deciphered
+from the json file containing the deployed data. Thus navigating to the
+``data_0.17.0`` folder and opening the ``deployment_services_goerli.json``
+shows us the following information.
+
++------------------+----------------------------------------------------------+
```
Comment (palango) [inline docs/index.rst]: Not sure if this really helpful, but if we leave it I'd recommend to make all addresses links to etherscan.

File: docs/index.rst (hunk 1 above)
Comment (agatsoh) [reply docs/index.rst]: Yes this does belong to the client about to remove it.

File: docs/index.rst (hunk 2 above)
Comment (agatsoh) [reply docs/index.rst]: Yes users exclusively of the MS and PFS services.

**Comment by agatsoh** [comment]
Main use of writing this is to present to the users of MS and PFS as much information as we can provide so the users are really clear. Still a work in progress.

File: docs/index.rst (hunk 3 above)
Comment (agatsoh) [reply docs/index.rst]: yes I will include the links to etherscan thanks for reminding

**Comment by palango** [comment]
@agatsoh Are you still working on this?

**Comment by palango** [comment]
@agatsoh Please reopen once you work on this again.

**Pull request closed without merging**